cfme/tests/cloud_infra_common/test_tag_objects.py

```diff
@@ -99,6 +101,27 @@ def _tagging_check(test_item, tag_place):
     return _tagging_check
 
 
+@pytest.fixture
+def tag_vm(provider, appliance):
+    """Get a random vm to tag"""
+    view = navigate_to(provider, 'ProviderVms')
+    all_names = view.entities.all_entity_names
+    return appliance.collections.infra_vms.instantiate(name=all_names[0], provider=provider)
+
+
+@pytest.fixture
+def tag_out_of_10k_values(appliance):
```

Comment:
Can you add a comment about what these commands are doing ? Rest all looks good :+1:

Reply:
added